content/02.intro.md

```diff
@@ -13,8 +13,9 @@ Across fields, "off-the-shelf" implementations of these algorithms have produced
 Deep learning approaches grew from research in neural networks, which were first proposed in 1943 [@doi:10.1007/BF02478259] as a model for how our brains process information.
 The history of neural networks is interesting in its own right [@doi:10.1103/RevModPhys.34.135].
 In neural networks, inputs are fed into the input layer, which feeds into one or more hidden layers, which eventually link to an output layer.
-A layer consists of a set of nodes, usually called "features" or "units," which are connected via edges to the immediately earlier and the immediately deeper layers.
-The nodes of the input layer generally consist of the variables being measured in the dataset of interest -- for example, each node could represent the intensity value of a specific pixel in an image processing application or a gene expression value in a transcriptomics experiment.
+A layer consists of a set of nodes, sometimes called "features" or "units," which are connected via edges to the immediately earlier and the immediately deeper layers.
+In some special neural network architectures, they can connect to themselves with a delay.
```

Comment:
Replace `they` with the specific word?  It may not be clear which noun you're referring to.